pkg/minishift/config/validations.go

```diff
@@ -121,3 +121,36 @@ func IsValidUrl(_ string, isoURL string) error {
 	}
 	return nil
 }
+
+func IsValidIPv4Address(name string, address string) error {
+	ip := net.ParseIP(address).To4()
+	if ip == nil {
+		return fmt.Errorf("'%s' is not a valid IPv4 address", address)
+	}
+
+	return nil
+}
+
+func IsValidNetmask(name string, mask string) error {
```

Comment:
@gbraad where is the `name` variable used for this method?

Reply:
fixed... although I think we should look critically at the text, as it now the correct use of English.